.github/scripts/execute_single_benchmark_wildcat.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# Datasets are stored in ~/sfm_datasets in order to reduce runtimes by not 
+# redownloading each (very large) dataset every CI run. Any new datasets must be
+# downloaded and stored in ~/sfm_datasets before running this action.
+DATASET_PREFIX=/home/tdriver6/sfm_datasets
```

Comment:
its not good practice to commit/merge paths. is there no other way of making this work?

Reply:
This is the actual path on the machine. It won't be different for different user

Reply:
I don't think there is without restructuring how we run the benchmarks. Either way, I think this is fine for now.